Common/Source/Dialogs/AddCustomKeyList.cpp

```diff
@@ -104,6 +104,11 @@ void AddCustomKeyList( DataField* dfe) {
     dfe->addEnumTextNoLF(MsgToken(2249)); // DspMode , not to be translated
     dfe->addEnumTextNoLF(MsgToken(2390)); // Toggle Draw XC
     dfe->addEnumTextNoLF(MsgToken(2337)); // Airspace lookup
+    dfe->addEnumTextNoLF(MsgToken(2251)); // Optimizer Mode
+    dfe->addEnumTextNoLF(MsgToken(2393)); // Device C Config
+    dfe->addEnumTextNoLF(MsgToken(2394)); // Device D Config
+    dfe->addEnumTextNoLF(MsgToken(2395)); // Device E Config
+    dfe->addEnumTextNoLF(MsgToken(2396)); // Device F Config
```

Comment:
that can't work, because "Optimizer mode" don't exist in custom key enum and CustomKeyLabel array...

Reply:
removed:
dfe->addEnumTextNoLF(MsgToken(2251)); // Optimizer Mode